main.cpp

```diff
@@ -66,25 +66,40 @@ int main()
 
 
 //1)
+
 struct Skateboard
 {
-    std::string gripTapeColor = "Green";
-    int wheelsHardness = 99;
-    std::string truckBrand = "Independent";
-    int bearingAgeYears = 2;
-    std::string woodType = "oak";
-  
+    std::string gripTapeColor; //= "Green";
+    int wheelsHardness; //= 99;
+    std::string truckBrand; //= "Independent";
+    int bearingAgeYears; //= 2;
+    std::string woodType; //= "oak";
+    Skateboard();
+
     float rollDownStreet(int speed); //returns how far you roll
     double railSlide(bool frontSide = true, double velocity = 10, double friction = 10); //returns the distance slid
     bool kickFlip(int degree = 360); //returns true if the kickflip was successfully performed.
 };
 
+Skateboard::Skateboard()
+{
+    gripTapeColor = "";
+    wheelsHardness = 0;
+    truckBrand = "";
+    bearingAgeYears = 0;
+    woodType = "";
```

Comment:
The gripTape has no color?  The trucks have no branding?  the wood has no type? 
- [ ] what kind of skateboard is this?